pkg/build/scanner.go

```diff
@@ -70,26 +70,26 @@ func Scanner(manifest model.Manifest, githubToken, git, branch string) error {
 		return fmt.Errorf("base image scan of %s failed. Unable to process exit code:\n %s", baseImageName, err.Error())
 	}
 
-	// Failed with an error indicating vulnerabilities were found. If IgnoreVulernability is true, just just return
-	if manifest.IgnoreVulnerability {
-		return nil
-	}
-
 	// Else build a new set of images.
-	// baseVersion is like 1.10-dev.1
-	// Increment the digit after the last period to get the new tag.
-	index := strings.LastIndex(baseVersion, ".") + 1
-	var digit int
-	if digit, err = strconv.Atoi(baseVersion[index:]); err != nil {
-		return err
+	buildTimestamp := strings.ReplaceAll(time.Now().Format(time.RFC3339), ":", "-")
+	log.Infof("new base tag: %s", buildTimestamp)
```

Comment:
doesn't need to be in this PR at all - but in the future with ephemeral containers we want `kubectl debug --image istio/base` to be usable for debugging. Having some way to say "also publish :latest tag" would be useful (which we can apply to the master branch).